trajectory_step_s: 0.008
servo_control_cycle: 0.02
init_x_offset: 0.0
init_y_offset: 0.0
init_z_offset: 0.015
init_roll_offset: 0
init_pitch_offset: 0
init_yaw_offset: 0
hip_pitch_offset: 20
period_time: 300
period_times: 0
dsp_ratio: 0.2
step_fb_ratio: 0.028
x_move_amplitude: 0.00
y_move_amplitude: 0.00
z_move_amplitude: 0.02
angle_move_amplitude: 0.0
y_swap_amplitude: 0.02
z_swap_amplitude: 0.006
pelvis_offset: 5
move_aim_on: False
arm_swing_gain: 0.5
